src/commands.json

```diff
@@ -58,7 +58,68 @@
         ],
         "since": "1.0.0",
         "group": "timeseries"
-        },
+    },
+    "DEL": {
```

Comment:
Why explicitly add a DEL when we have TS.DEL?

Reply:
@chayim It can delete multiple series/key at once